File: Ejercicio_2.py
import psycopg2

while True:

    conexion = psycopg2.connect(
        host = "localhost",
        port = "5432",
        user = "postgres",
        password = "12345abc",
        dbname = "Tarea_Preparatoria"
    )

    print("""
    1) Abrir programa
    2) Ver el historial""")

    nn1 = int(input("Seleccione una opcion: "))

    if nn1==1:

        a = int(input("Primer angulo: "))
        b = int(input("Segundo angulo: "))

        c = 180-a-b
        print("El angulo es: ",c)

        cursor = conexion.cursor()
        cursor.execute("INSERT INTO ejer2(primer_angulo,segundo_angulo,angulo_resultante) VALUES(%s,%s,%s);",(a,b,c))
        conexion.commit()
        cursor.close()
        conexion.close()

    elif nn1==2:
        cursor = conexion.cursor()
        SQL = "SELECT*FROM ejer2;"
        cursor.execute(SQL)
        valores = cursor.fetchall()
        print(valores)
        cursor.close()
        conexion.close()

File: Ejercicio__2.m
clc;clear;

try
	fprintf('Determinar el angulo restante de un triangulo \n');
	a = input('Ingrese el primer angulo: ');
  b = input('Ingrese el segundo angulo: ');
  c = 180-a-b;
  pkg load database
  conn = pq_connect(setdbopts('dbname','Tarea_Preparatoria','host','localhost','port','5432','user','postgres','password','12345abc'));

    printf('El angulo es igual a %i \n', c);
		concat=strcat('El angulo flatante es c=', num2str(c));
		N = pq_exec_params(conn,"insert into ejer2(primer_angulo, segundo_angulo, angulo_resultante) values($1,$2,$3);",{a,b,c});
	
catch
	printf("Ha ocurrido un error \n");
end_try_catch
